File: java/socketd/src/main/java/org/noear/socketd/transport/core/MessageInternal.java
package org.noear.socketd.transport.core;

/**
 * 消息内部接口
 *
 * @author noear
 * @since 2.0
 */
public interface MessageInternal extends Message, Reply {
    /**
     * 获取标记
     */
    int flag();
}

File: swift/socketd/socketd/src/socketd/transport/core/MessageInternal.swift
//
//  MessageInternal.swift
//  socketd
//
//  Created by noear on 2023/12/27.
//

import Foundation

/**
 * 消息内部接口
 *
 * @author noear
 * @since 2.0
 */
protocol MessageInternal : Message, Reply{
    /**
     * 获取标记
     */
    func flag() -> Int32;
}
